src/controllers/usersController.js

```diff
@@ -0,0 +1,38 @@
+import UsersService from '../services/usersService';
+
+/**
+ * Class for interacting with User profile information
+ */
+export default class UsersController {
+/**
+ * @param {object} user - user auth token payload
+ * @param {object} res - response object
+ * @return {object} user - return object containing status and data
+ */
+  static async getProfile({ user }, res) {
+    try {
+      const { id } = user;
+      const userData = await UsersService.getProfile(id);
+
+      res.status(200).json({ status: 'success', data: userData });
```

Comment:
there's a utility function for sending responses in the utils folder, please utilizes it